testing/trino-testing-services/src/main/java/io/trino/jmh/Benchmarks.java

```diff
@@ -65,9 +66,24 @@ public BenchmarkBuilder withOptions(Consumer<ChainedOptionsBuilder> optionsConsu
             return this;
         }
 
+        public BenchmarkBuilder includeAll()
+        {
+            optionsBuilder.include("^\\Q" + benchmarkClass.getName() + ".\\E");
+            return this;
+        }
+
+        public BenchmarkBuilder includeMethod(String benchmarkMethod)
+        {
+            optionsBuilder.include("^\\Q" + benchmarkClass.getName() + ".\\E" + benchmarkMethod + "$");
```

Comment:
wrap `benchmarkMethod` in `"(" ... ")"` -- since it's regex, it could contain eg `|`

btw the fact it's regex is useful, it allows a set of methods to be run

Reply:
it's probably better to use multiple `includeMethod` calls instead of `|` operator but I added the brackets for it to work as expected anyway